### PR TITLE
Added My Username

### PR DESCRIPTION
**Pull request by criticic**
Issue: 1

#### Short description of what this resolves:
Added my name.
Screenshot from 2021-10-04 01-11-46

### CODE REVIEW

**Comment by criticic** [comment]
@Srujana2505 @Mitul16 Please take a look.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1 +1,2 @@
 ## Contributors
+cynik
```
Comment (Mitul16) [inline CONTRIBUTORS.md:2]: This doesn't seem to be your correct GitHub username. Isn't it `criticic`?

Comment (criticic) [reply CONTRIBUTORS.md:2]: I use both, depending on availability, I'll fix it.

**Comment by criticic** [comment]
@Mitul16 i have resolved the conflicts and corrected my username.

**Pull request merged**